src/Aspire.Hosting.Azure.Redis/AzureRedisEnterpriseExtensions.cs

```diff
@@ -1,7 +1,8 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-using System.Diagnostics.CodeAnalysis;
+#pragma warning disable AZPROVISION001 // Type is for evaluation purposes only and is subject to change or removal in future updates. Suppress this diagnostic to proceed.
```

Comment:
I opened [Remove Experimental AZPROVISION001 from Azure.Provisioning.RedisEnterprise (Azure/azure-sdk-for-net#54062)](https://github.com/Azure/azure-sdk-for-net/pull/54062) for this.